integration-cli/docker_cli_build_test.go

```diff
@@ -4375,6 +4375,27 @@ func (s *DockerSuite) TestBuildTimeArgHistoryExclusions(c *check.C) {
 	}
 }
 
+// Reported in moby/moby#32744
+func (s *DockerSuite) TestBuildTimeArgHistoryExclusionsMultipleBuiltins(c *check.C) {
+	imgName := "bldargtest"
+	dockerfile := fmt.Sprintf(`FROM busybox
+		RUN echo "Testing Build Args!"`)
+	buildImage(imgName,
+		cli.WithFlags("--build-arg", "http_proxy=foo",
+			"--build-arg", "https_proxy=bar",
+		),
+		build.WithDockerfile(dockerfile),
+	).Assert(c, icmd.Success)
```

Comment:
You could argue that it's not just testing the oob error but that it's testing the use case...
If you are against having a dedicated test for it, I can collapse it in to the previous build-arg exclusions test.
As for adding a unit test, I'll take a look and see how easy that'll be...